### DISCUSSION TITLE
Uranium UI controls on Uno Platform apps

### DISCUSSION THREAD

**Discussion by sasakrsmanovic** [enisn/UraniumUI#440] [General]
Hello

Last week we introduced MAUI Embedding feature to Uno Platform apps. This should mean that Uranium UI controls can now work on Uno Platform apps too. 

https://platform.uno/docs/articles/external/uno.extensions/doc/Overview/Maui/MauiOverview.html

Would anyone be interested to work with us to test Uranium UI controls on it and contribute a getting started similar to what we have for i.e. MAUI Community toolkit https://platform.uno/docs/articles/external/uno.extensions/doc/Overview/Maui/ThirdParty-MauiCommunityToolkit.html

Thanks!

**Comment by enisn** [comment]
That is awesome 🎉

 I'll create a simple getting-started when I find some time to do it

**Reply by nickrandolph** [reply]
Great. Reach out to @sasakrsmanovic  or myself if you need any assistance getting your controls to work.
